### PR TITLE
Cloudstack quota test refactor pr two

### PR DESCRIPTION
**Pull request by fgan1**
# Description
Refactoring the Cloudstack Compute Quota Plugin context.

# Proposed changes
- Implementing tests to the Request/Response Cloudstack.

# Additional information
- This branch depends on a branch in the Fogbow Common (cloudstack-quota-test-refator)

# PR Slices
[x] One: Refactoring Cloudstack Compute Quota Plugin.
[x] Two: Implementing tests to the Request/Response Cloudstack.
[] Three: Refactoring Cloudstack Compute Quota Plugin Tests.

### CODE REVIEW

File: src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackCloudUtils.java [hunk 1]
```diff
@@ -43,4 +53,24 @@ public static String doRequest(@NotNull CloudStackHttpClient cloudStackHttpClien
         }
     }
 
+    // TODO(chico) - Implement tests
```
Comment (pauloewerton) [inline src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackCloudUtils.java]: does this todo still hold?

File: src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "listresourcelimitsresponse": {
+    "resourcelimit": [
+      {
+        "domainid": "%s",
+        "resourcetype": "%s",
+        "max": %s
```
Comment (pauloewerton) [inline src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse.json]: nit: missing quotation marks?

File: src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse_error.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "listresourcelimitsresponse": {
+    "errorcode":%s,
```
Comment (pauloewerton) [inline src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse_error.json]: ditto

File: src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackCloudUtils.java (hunk 1 above)
Comment (fgan1) [reply src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackCloudUtils.java]: I removed this TODO in the third PR.

File: src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse.json (hunk 2 above)
Comment (fgan1) [reply src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse.json]: It is not missing. The "max" is an integer.

File: src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse_error.json (hunk 3 above)
Comment (fgan1) [reply src/test/resources/cloud/plugins/interoperability/cloudstack/listresourcelimitsresponse_error.json]: The "errorcode" is an integer.

**Pull request merged**